src/connection.rs

```diff
@@ -308,6 +308,25 @@ impl rustls_connection {
         }
     }
 
+    /// Queues a TLS1.3 key_update message to refresh a connection’s keys.
+    ///
+    /// Rustls internally manages key updates as required and so this function should
+    /// seldom be used. See the Rustls documentation for important caveats and suggestions
+    /// on occasions that merit its use.
+    ///
+    /// <https://rustls.github.io/rustls/prerelease/client/struct.ClientConnection.html#method.refresh_traffic_keys>
```

Comment:
How about a permalink here: https://docs.rs/rustls/0.23.14/rustls/struct.ConnectionCommon.html#method.refresh_traffic_keys

Reply:
(and below)

Reply:
@ctz Oops didn't realize I was putting prerelease links in there. Will fix.

> (and below)

You mean the generated instance in `rustls.h`? I'm not sure I spot another bad link outside of L317 in `connection.rs` and the `rustls.h` instance

Reply:
Ah yes I was referring to the cbindgen output in rustls.h, lol

Reply:
Fixed. I used a `/latest` link instead of `0.23.14` since it was consistent with the other URLs in-file.